frontend/common/src/constants/localizedConstants.tsx

```diff
@@ -649,14 +657,75 @@ export const operationalRequirements = defineMessages({
   },
 });
 
+export const operationalRequirementLabelShort = defineMessages({
```

Comment:
These are the short strings used in the fancy filter with limited space. Will they suffice as the v1 labels, comparable to this:
https://github.com/GCTC-NTGC/gc-digital-talent/pull/3388/files#diff-fb251ec51456fdc6fbe09424d4f94b19ecb7dceac37971e853209b885bac2147R661-R699

If not, we could either make another label variant, or you can suggest the important changes, and we'll just have "not-quite-the-shortest" values for the fancy filter :)

Reply:
> maybe those messages could be defined in the component instead of the general localizedConstants?

storing label variants in components admittedly goes pretty counter to how I was hoping to eventually consolidate into an option builder hook 🤔 

I don't think I fully understand the rationale for why these variants can't be merged when they're so close, but I'll assume the text is very dialled in, and I'll just create a new label variant set in the fancy filter PR, or just use the slightly longer text as-is :)

Reply:
Ah, just now realized you'd already done this...!